lib/bolt/transport/winrm/connection.rb

```diff
@@ -41,7 +41,7 @@ def connect
 
           transport = :kerberos if target.options['realm']
           endpoint = "#{scheme}://#{target.host}:#{@port}/wsman"
-          cacert = target.options['cacert'] && target.options['ssl'] ? target.options['cacert'] : nil
+          cacert = target.options['cacert'] && target.options['ssl'] ? File.expand_path(target.options['cacert']) : nil
```

Comment:
I went ahead and added this commit on this PR because it build on the new logic here. See https://github.com/puppetlabs/bolt/pull/1233 for similar fix for orch transport.